week-1/L-functions-nested/exercise2.js

```diff
@@ -1,5 +1,17 @@
+function greeting() {
+  return "HELLO";
+}
+function upperCase(name) {
+  return name.toUpperCase();
+}
+
 var mentor1 = "Daniel";
 var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+console.log(upperCaseNames(mentor1));
+console.log(upperCaseNames(mentor2));
+console.log(upperCaseNames(mentor3));
+console.log(upperCaseNames(mentor4));
+console.log(upperCaseNames(mentor5));
```

Comment:
It looks like you forgot to declare a function named `upperCaseNames` which combines `greeting` and `upperCase`.

Reply:
@bonboh Tanks for your feedback and suggestions. I will correct my mistakes on a few exercises.